modules/libarchive/3.7.9/overlay/libarchive/test/BUILD.bazel

```diff
@@ -0,0 +1,917 @@
+libarchive_test(name = "test_open_filename", deps = [":lib"])
+libarchive_test(
+    name = "test_pax_filename_encoding",
+    deps = [":lib"],
+    tests = [
+        "test_pax_filename_encoding_KOI8R",
+        "test_pax_filename_encoding_CP1251",
+        "test_pax_filename_encoding_EUCJP",
+        "test_pax_filename_encoding_CP932",
+        "test_pax_filename_encoding_KOI8R_BINARY",
+        "test_pax_filename_encoding_KOI8R_CP1251",
+        "test_pax_filename_encoding",
+    ],
+)
+libarchive_test(name = "test_pax_xattr_header", deps = [":lib"])
+libarchive_test(name = "test_read_data_large", deps = [":lib"])
+libarchive_test(name = "test_read_disk", deps = [":lib"])
+libarchive_test(name = "test_read_disk_directory_traversals", deps = [":lib"])
+libarchive_test(name = "test_read_disk_entry_from_file", deps = [":lib"])
+libarchive_test(name = "test_read_extract", deps = [":lib"])
+libarchive_test(name = "test_read_file_nonexistent", deps = [":lib"])
+libarchive_test(
+    name = "test_read_filter_compress",
+    deps = [":lib"],
+    tests = [
+        "test_read_filter_compress_truncated",
+        "test_read_filter_compress_empty2",
+        "test_read_filter_compress_invalid",
+    ],
+)
+libarchive_test(name = "test_read_filter_grzip", deps = [":lib"])
+libarchive_test(name = "test_read_filter_lrzip", deps = [":lib"])
+libarchive_test(name = "test_read_filter_lzop", deps = [":lib"])
+libarchive_test(name = "test_read_filter_lzop_multiple_parts", deps = [":lib"])
+libarchive_test(name = "test_read_filter_program", deps = [":lib"])
+libarchive_test(name = "test_read_filter_program_signature", deps = [":lib"])
+libarchive_test(
+    name = "test_read_filter_uudecode",
+    deps = [":lib"],
+    tests = [
+        "test_read_filter_uudecode",
+        "test_read_filter_uudecode_base64",
+    ],
+)
+libarchive_test(
+    name = "test_read_filter_uudecode_raw",
+    deps = [":lib"],
+    tests = [
+        "test_read_filter_uudecode_raw",
+        "test_read_filter_uudecode_base64_raw",
+    ],
+)
+libarchive_test(
+    name = "test_read_format_7zip",
+    deps = [":lib"],
+    tests = [
+        "test_read_format_7zip",
+        "test_read_format_7zip_bzip2",
+        "test_read_format_7zip_from_fd",
+        "test_read_format_7zip_copy",
+        "test_read_format_7zip_deflate",
+        "test_read_format_7zip_zstd",
+        "test_read_format_7zip_zstd_solid",
+        "test_read_format_7zip_zstd_bcj",
+        "test_read_format_7zip_zstd_nobcj",
+        "test_read_format_7zip_empty",
+        "test_read_format_7zip_lzma1",
+        "test_read_format_7zip_lzma2",
+        "test_read_format_7zip_zstd_arm",
+        "test_read_format_7zip_lzma2_arm",
+        "test_read_format_7zip_ppmd",
+        "test_read_format_7zip_lzma2_arm64",
+        "test_read_format_7zip_deflate_arm64",
+        "test_read_format_7zip_win_attrib",
+    ],
+    target_compatible_with = select({
+        # TODO(zaucy?): make it pass on windows
```

Comment:
FYI @zaucy this is the one change I had to make from previous version to get BCR presubmit green

Reply:
thanks for the heads up - not sure what the problem is currently